django_gcloud_storage/__init__.py

```diff
@@ -107,7 +107,7 @@ def close(self):
 @deconstructible
 class DjangoGCloudStorage(Storage):
 
-    def __init__(self, project=None, bucket=None, credentials_file_path=None):
+    def __init__(self, project=None, bucket=None, credentials_file_path=None, unsigned_urls=None):
```

Comment:
I suggest renaming this parameter to use_unsigned_urls to make it more clear that this is a boolean flag.